basicview.cpp

```diff
@@ -18,38 +18,77 @@ limitations under the License.
 
 #include <QVBoxLayout>
 
+#include <QFontDatabase>
+#include <QFont>
+#include <QDebug>
+#include <QChar>
+
+const QString startIcon = "\uf04b";
+const QString stopIcon = "\uf04d";
+const QString pauseIcon = "\uf04c";
+const QString deleteIcon = "\uf1f8";
+const QString reloadIcon = "\uf021";
+
+void BasicView::setFA(QWidget *wid)
+    {
```

Comment:
Formatting seems off

Reply:
thats actually what the formmater did for me, it makes the curly brace in next line, when I run
clang-format -i *.cpp *.h